indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -144,6 +145,14 @@ def _process(self):
                 weight=80,
             ))
 
+        if event.has_feature('payment'):
+            action_menu_items.append(ActionMenuEntry(
+                _('Update Registration Fee'),
+                'coins',
+                url=url_for('.registrations_base_price', regform),
+                weight=40
+            ))
```

Comment:
There is a small issue with this. When a registration's state changes, it doesn't update the list, but if I set `reload_page=True`, the flash messages don't get displayed... Is there a simple way around this?

Reply:
do you use jsonify-response? if yes pass the argument to not include flashed in that response

Reply:
aaah I see. that works, thanks!